### DISCUSSION TITLE
PDF page count

### DISCUSSION THREAD

**Discussion by jaka1-newbie** [dlemstra/Magick.NET#1062] [Help] [answered]
```
using ImageMagick.Formats.Pdf;

var pdfInfo = PdfInfo.Create(@"C:\YourFile.pdf");

```

This don't work for me.
Imagemagick don't contain definition for pdf

**Comment by dlemstra** [comment]
What doesn't work? And did you install Ghostscript on your machine?

**Comment by jaka1-newbie** [comment]
Компиляция кода  Ошибка в действии "CS0234" "Ошибка в директивах using. The type or namespace name 'Formats' does not exist in the namespace 'ImageMagick' (are you missing an assembly reference?)". [Строка: 7; Cтолбец: 19]

**Comment by jaka1-newbie** [comment]
GhostScript installed

**Comment by dlemstra** [comment, marked as answer]
You should be using `using ImageMagick.Formats` instead.

**Comment by jaka1-newbie** [comment]
Тип	Время	Сообщение
	15:38:17	Компиляция кода проекта Ошибка при компиляции общего кода "CS0234" "The type or namespace name 'Formats' does not exist in the namespace 'ImageMagick' (are you missing an assembly reference?)". [Строка: 23; Cтолбец: 19]

**Comment by jaka1-newbie** [comment]
My Version of Magick.Net
![image](https://user-images.githubusercontent.com/67605008/138559265-166e6f3e-a7bf-4c70-9dc0-e4e9082d6f4f.png)

I cant add working version of Magick.Net 8.3.3
